applications/plugins/image/ImageContainer.h

```diff
@@ -547,36 +547,20 @@ class ImageContainer : public core::objectmodel::BaseObject
 
     void draw(const core::visual::VisualParams* vparams) override
     {
-#ifndef SOFA_NO_OPENGL
         // draw bounding box
-
         if (!vparams->displayFlags().getShowVisualModels()) return;
         if (!drawBB.getValue()) return;
 
-        glPushAttrib( GL_LIGHTING_BIT | GL_ENABLE_BIT | GL_LINE_BIT );
-        glPushMatrix();
+        vparams->drawTool()->saveLastState();
 
-        const float color[]={1.,0.5,0.5,0.}, specular[]={0.,0.,0.,0.};
-        glMaterialfv(GL_FRONT_AND_BACK,GL_AMBIENT_AND_DIFFUSE,color);
-        glMaterialfv(GL_FRONT_AND_BACK,GL_SPECULAR,specular);
-        glMaterialf(GL_FRONT_AND_BACK,GL_SHININESS,0.0);
-        glColor4fv(color);
-        glLineWidth(2.0);
+        float color[]= {1.,0.5,0.5,0.};
+        vparams->drawTool()->setMaterial(color);
```

Comment:
To solve the compilation error, 
`vparams->drawTool()->setMaterial( {1.,0.5,0.5,0.} );`